src/Uno.UWP/ApplicationModel/DataTransfer/DataPackage.cs

```diff
@@ -93,6 +93,146 @@ public DataPackageView GetView()
 				_data,
 				ResourceMap.ToImmutableDictionary(),
 				(id, op) => OperationCompleted?.Invoke(this, new OperationCompletedEventArgs(id, op)));
+
+		/// <summary>
+		/// Determines if the given URI/URL is considered a WebLink.
+		/// This determination is done based on whether the scheme starts with 'http' or 'https'.
+		/// </summary>
+		/// <remarks>
+		/// This method is intended for use during integration with other platforms.
+		/// Therefore, it uses a string instead of a Uri or native URL class.
+		/// </remarks>
+		/// <param name="uri">The URI to determine if it is a WebLink.</param>
+		/// <returns>True if the given URI/URL is considered a WebLink; otherwise, false.</returns>
+		internal static bool IsUriWebLink(string? uri)
```

Comment:
Codacy found an issue: [Either change this parameter type to 'System.Uri' or provide an overload which takes a 'System.Uri' parameter.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6364607)